gradle.properties

```diff
@@ -1,7 +1,7 @@
 version=1.6.0-RC1-observers-SNAPSHOT
 kotlin_version=1.8.21
 dokka_version=1.8.10
-antlr_version=4.9.3
+antlr_version=4.12.0
```

Comment:
This will make Java 11+ a requirement, not that it's wrong but we have to be aware of that

Reply:
Right, let's revert that